pkg/networks/gvisor.go

```diff
@@ -43,7 +43,7 @@ func StartGVisorNetstack(ctx context.Context, gVisorOpts *GVisorNetstackOpts) {
 		Debug:             false,
 		MTU:               opts.MTU,
 		Subnet:            SlirpNetwork,
-		GatewayIP:         SlirpGateway,
+		GatewayIP:         SlirpDNS,
```

Comment:
Could you add a code comment to explain this

Reply:
Done. Hope this is clear now.

Reply:
The convention in Go for code comments is to have a space between the double slash `//` and the text.

Reply:
I have to admit that even with the comment, I still don't understand **why** this works. Why is it called a gateway if it isn't actually a gateway?

Reply:
I think the constants specific to vz driver should be re-defined to avoid confusion. Type aliases can be used.

Reply:
Yes, thanks, LGTM when the above is added as code comment lines

Reply:
Done

Reply:
Done